### PR TITLE
The basic implementation of the app

### PR DESCRIPTION
**Pull request by kanishkarj**
The basic implementation of the app

### CODE REVIEW

**Comment by kanishkarj** [comment]
@nveenjain @leecalcote

File: service/main.go [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+		http.Error(w, "Method not defined", http.StatusBadRequest)
+	}
+}
+
+func main() {
+	logrus.SetOutput(os.Stdout)
+
+	mux := http.NewServeMux()
+	mux.Handle("/call", MetricsMiddleware(http.HandlerFunc(call)))
+	mux.Handle("/metrics", http.HandlerFunc(metrics))
```
Comment (leecalcote) [inline service/main.go:123]: Does this use the Prometheus exposition format (is this Prometheus compatible)?

Comment (kanishkarj) [reply service/main.go:123]: I don't know, I didn't consider prometheus while implementing this. I could make it compatible if it is required.

Comment (leecalcote) [reply service/main.go:123]: Ok. We may find that it is helpful in the future.

**Comment by leecalcote** [comment]
Perhaps, @naturalett might work his GitHub Actions magic on a build workflow for this app.

File: service/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,5 @@
+FROM golang:1.13.5 as bd
+WORKDIR /github.com/layer5io/sample-app-service
+ADD . .
+RUN GOPROXY=direct GOSUMDB=off go build -a -o /main .
+CMD ["/main"]
```
Comment (nveenjain) [inline service/Dockerfile:5]: Should we compile it to ./main instead of root directory? Thoughts @kanishkarj ?

Comment (kanishkarj) [reply service/Dockerfile:5]: Oh yes, that would be better. I'll made that change!

**Pull request merged**